### PR TITLE
rename dashboard to selfservice

### PR DESCRIPTION
**Pull request by JiggyPete**
Currently we have a `/dashboard` as the root url namespace, 
all controllers have that namespace, 
`dashboard.en.yml` contains all the translations
and there are a number of scss files relating to this work.

This PR is to rename all of this, so that the routes now look `/selfservice`
and all controllers have the `SelfService` namespace,
translations are now in self_service.en.yml
and scss classes now refer to `self-service`.

### CODE REVIEW

File: app/assets/stylesheets/page_specific/_self_service.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.dashboard__action-button {
+.self -service__action-button {
```
Comment (neoeno) [inline app/assets/stylesheets/page_specific/_self_service.scss]: ;)

File: config/routes.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@
     resources :advisers, only: :show
   end
 
-  namespace :dashboard do
+  namespace :self_service, path: 'selfservice' do
```
Comment (neoeno) [inline config/routes.rb]: why the squashed up path?

Comment (JiggyPete) [inline config/routes.rb]: I figured that was what MAS would want. It's been updated as per rails default self_service

**Comment by neoeno** [comment]
:+1:

**Comment by rnalexander** [comment]
:+1: :shipit:

**Pull request merged**